### PR TITLE
Add command line tool to run tests

### PR DESCRIPTION
**Pull request by arnaud-tincelin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,46 @@
...
+name: Name of the test
+cases:
+- variables:
+  - key: variable name in TF
+    value: val
+  wantErr: true
+- variables:
+  - key: variable name in TF
+    value: val2
+  wantErr: false
```
Comment (sdib) [inline README.md]: instead of wantErr I'd propose errorExpected, what do you think ?

File: .github/workflows/PolicyTesting.yml [hunk 2]
```diff
@@ -1,91 +1,41 @@
...
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ env.GO_VERSION }}
     - name: Checkout
       uses: actions/checkout@v2
-
-    # Install the latest version of Terraform CLI and configure the Terraform CLI configuration file with a Terraform Cloud user API token
-    - name: Setup Terraform
-      uses: hashicorp/setup-terraform@v1
+    - name: Go tests
```
Comment (sdib) [inline .github/workflows/PolicyTesting.yml]: before testing I'd add a step to check formatting of go and terraform files

for example equivalent in AzDO pipeline : 

``` yaml
- pwsh: |
      $formattedFiles = Invoke-Expression -Command "go fmt ./..."
      $nbFiles = ($formattedFiles | Measure-Object -Line).Lines
      if ($nbFiles -ne 0) {
          Write-Output "##vso[task.logissue type=error]Following files need to be linted with 'go fmt': $formattedFiles"
          Write-Output "##vso[task.complete result=Failed;]"
      } else {
          Write-Output "##vso[task.complete result=Succeeded;]"
      }
    displayName: Check Go formatting
```

``` yaml
- pwsh: |
      $formattedFiles = Invoke-Expression -Command "terraform fmt --recursive"
      $nbFiles = ($formattedFiles | Measure-Object -Line).Lines
      if ($nbFiles -ne 0) {
          Write-Output "##vso[task.logissue type=error]Following files need to be linted with 'terraform fmt': $formattedFiles"
          Write-Output "##vso[task.complete result=Failed;]"
      } else {
          Write-Output "##vso[task.complete result=Succeeded;]"
      }
    displayName: Check Terraform formatting
```

File: README.md (hunk 1 above)
Comment (arnaud-tincelin) [reply README.md]: done

**Review by sdib — APPROVED** [review]
LGTM

**Review by jerecarl — APPROVED** [review]

**Pull request merged**